### PR TITLE
Add check to signing builds for strong name signature

### PR DESCRIPTION
**Pull request by cormacpayne**
Fix for issue https://github.com/Azure/azure-powershell/issues/2762

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -258,6 +258,8 @@
         ContinueOnError="false" 
         Condition="!$(DelaySign) and '@(DelaySignedAssembliesToSign)' != ''"/>
         
+    <Exec Command="$(MSBuildProjectDirectory)\tools\CheckStrongNameSignature.ps1"/>
```
Comment (markcowl) [inline build.proj]: See: https://github.com/cormacpayne/azure-powershell/blob/5043c685511c5fc8d678a4e245c9bdfa41ad83ff/build.proj#L308  for how to run powershell and refer to the tools folder in a task

Comment (shahabhijeet) [inline build.proj]: @cormacpayne any reason we need the ps1 file, rather than adding the same code in the ValidateStrongNameSignatureTask ?

**Comment by markcowl** [comment]
signing build here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/580/

**Comment by cormacpayne** [comment]
Signed build: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/581/

**Pull request merged**